### PR TITLE
Add release notes for v0.8.0

### PR DESCRIPTION
**Pull request by mrsarm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,19 @@ The medic-android application is a thin wrapper to load the [CHT Core Framework]
...
 
-### 0.7.3
+## 0.8.0
+
+### Changes
+
+- [improvement] [#134](https://github.com/medic/medic-android/issues/134) New connection errors UX:
+  - The improvements only apply to _Webview_ flavors.
+  - It also applies when the app migrates to Webview from a XWalk installation.
+- [improvement] [#163](https://github.com/medic/medic-android/issues/163) New UX of Crosswalk to Webview migration:
```
Comment (craig-landry) [inline README.md:15]: You have the issue numbers and descriptions mismatched here.

134 = `Improve UX of crosswalk to webview migration`

163 = `Improve connection errors UX, especially for the XWalk-Webview migration process`

**Review by craig-landry — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mrsarm) [reply README.md]: :facepalm: fixed.

File: README.md [hunk 2]
```diff
@@ -5,7 +5,19 @@ The medic-android application is a thin wrapper to load the [CHT Core Framework]
...
+## 0.8.0
+
+### Changes
+
+- [improvement] [#163](https://github.com/medic/medic-android/issues/163) New connection errors UX:
+  - The improvements only apply to _Webview_ flavors.
+  - It also applies when the app migrates to Webview from a XWalk installation.
+- [improvement] [#134](https://github.com/medic/medic-android/issues/134) New UX of Crosswalk to Webview migration:
+  - Add splash screen while the data is migrated.
+  - Fix bug that cause redirect to the login page after migrate.
```
Comment (craig-landry) [inline README.md:17]:
```suggestion
  - Fix bug that caused redirect to the login page after migrate.
```

**Review by craig-landry — CHANGES_REQUESTED** [review]
Sorry I missed a little grammar thing in my initial review. After that, this looks good to go.

**Review by craig-landry — APPROVED** [review]

**Pull request merged**